src/main/kotlin/com/openlattice/mechanic/upgrades/MaterializationForeignServer.kt

```diff
@@ -52,7 +52,7 @@ class MaterializationForeignServer(
 
     private fun updateForeignServerPort(organizationId: UUID) {
         val organizationDbName = PostgresDatabases.buildOrganizationDatabaseName(organizationId)
-        connect(organizationDbName).use { dataSource ->
+        connect(organizationDbName).let { dataSource ->
```

Comment:
Since this update is going through each organization once and we don't cache the connection, I think we should use `use` and close the connection.

Reply:
Agreed.